src/format/tests.rs

```diff
@@ -349,8 +349,8 @@ pub external type Four
     assert_format!(
         r#"fn main() {
   fn() {
-    1
-    2
+    "1"
+    "2"
```

Comment:
Looks like we're now formatting Ints as strings too. If we remove the `.surround("\"", "\"")` from `src/format.rs` for `UntypedExpr::Int` and `Pattern::Int` then it should fix it I think :)

Reply:
oh, sorry formatter was stronger then me then me this time ;) fixed